src/main/scala/tech/cryptonomic/conseil/process/AccountsTaggingProcess.scala

```diff
@@ -0,0 +1,157 @@
+package tech.cryptonomic.conseil.process
+import cats.Applicative
+import cats.effect.Sync
+import cats.syntax.applicative._
+import cats.syntax.flatMap._
+import cats.syntax.functor._
+import cats.syntax.option._
+import cats.instances.int._
+import cats.mtl.MonadState
+import scala.{Stream => _} //hide from default scope
+import fs2._
+import tech.cryptonomic.conseil.tezos.Tables.OperationsRow
+import tech.cryptonomic.conseil.tezos.TezosTypes.PublicKeyHash
+import tech.cryptonomic.conseil.util.PureLogging
+
+object AccountsTaggingProcess {
+
+  /** Creates an instance of the process, which is an effectful operation */
+  def apply[F[_]: Sync: MonadState[?[_], Option[BlockLevel]]]: AccountsTaggingProcess[F] =
+    new AccountsTaggingProcess[F]
+
+  /** Convenience alias */
+  type BlockLevel = BigDecimal
+
+  /** The kinds of relevant operations for tagging */
+  val operationKinds = Set("activate_account", "reveal")
+
+}
+
+/** Scans the existing blocks stored by conseil for specific account-related operations
+  * i.e. Revelation, Activation.
+  * It then marks such accounts with the appropriate flag (e.g. revealed, activated).
+  * @param highWatermarkRef a pure reference that tracks the highest verified block level for flagging operations
+  */
+class AccountsTaggingProcess[F[_]: Sync] private ( //highWatermarkRef: Ref[F, Option[AccountsTaggingProcess.BlockLevel]])(
```

Comment:
Dead comment

Reply:
Undead stuff... I thought I'd removed this too

Nice one